api/requests/get_accounts.go

```diff
@@ -0,0 +1,18 @@
+package requests
+
+import (
+	ethCommon "github.com/ethereum/go-ethereum/common"
+	"github.com/hermeznetwork/hermez-node/common"
+	"github.com/iden3/go-iden3-crypto/babyjub"
+)
+
+// GetAccountsAPIRequest is an API request struct for getting accounts
+type GetAccountsAPIRequest struct {
```

Comment:
I see that you created a new file for each `Request` type.  I think this introduces too many files and makes it a bit harder to navigate the code.  Here are my suggestions (what I would have done differently):
1. I would declare the types in the same file as the function that uses them.  For each API function, just before it declare the `Request` type used for that function.  I don't understand the need to split them into files, and I don't understand the need to move them to another package.

Reply:
I agree. 
I don't think we would need to create another package and I also Go source files standard don't use _ between the names, except for the test files. 
I would suggest to include all the structs into parameter.go file and within same package.

Reply:
@jeffprestes and @ed255 thank you for the suggestions! But what I should do, if placing those structs in the same package will cause cyclical dependence?